docs/playbooks/intro/index.ikb

```diff
@@ -12,3 +12,4 @@ events         : events/index.ikb
 
 aws : aws/index.ikb
 gcp : gcp/index.ikb
+mac : macosx/index.ikb
```

Comment:
`macosx` is a little odd. I'd either call it `darwin` (as in GOOS) or `macos` (as in the new official name macOS)